nav_msgs/OccupancyGrid map
std_msgs/String G_json_str
---
nav_msgs/Path path
